build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -404,6 +413,10 @@ def populateSettingValuesFromConfigMap(parsedConfig)
     file.write("export MONITORING_MAX_EVENT_RATE=#{@mdsdMonitoringMaxEventRate}\n")
   end
 
+  if @mdsdBackPressureThreshold > 0
+    file.write("export BACKPRESSURE_THRESHOLD=#{@mdsdBackPressureThreshold}\n")
```

Comment:
include units in the name of this variable BACKPRESSURE_THRESHOLD as well.

Reply:
done